### ISSUE TITLE
API To Serve JSON

### ISSUE THREAD

**Issue by Ezequiel-Valencia** [AvocadoMoon/Events#1]
### Problem
Every time a JSON file name changes, or is moved every reference to it by a direct URL has to change.

### Solution
Create a static site which acts as an API strictly for grabbing JSON only, becoming a sort of interface.

**Comment by Ezequiel-Valencia** [comment]
[Requirements for Static Site](https://www.reddit.com/r/webdev/comments/1emwoz7/whats_the_minimal_github_action_for_hosting_a/)

**Comment by Ezequiel-Valencia** [comment]
The static site representation by Github requires HTML, CSS, and Javascript files to be severed. Folders acting as paths within the URL. The router that would provide indirection from file names seems to be impossible without a separate tool that would build provided files into this HTML, CSS, and Javascript that would be required | [Jeykll]() | [Ruby]()

### TLDR
Does not seem worth setting up a build system for serving the static JSON.
